lib/wrapbox/runner/ecs.rb

```diff
@@ -97,6 +97,7 @@ def run(class_name, method_name, args, container_definition_overrides: {}, **par
       def run_cmd(cmds, container_definition_overrides: {}, **parameters)
         task_definition = prepare_task_definition(container_definition_overrides)
 
+        cmds << "" if cmds.empty?
```

Comment:
[imo] I think `cmds << nil` is better because it doesn't seem to override `command` and allows users to execute the default command of Docker images.